### PR TITLE
Create PlusOneTest.java

### PR DESCRIPTION
**Pull request by adjamalbek**
added j-unit test for problem

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Comment by adjamalbek** [comment]
Added Junit Tests. Kindly review it @siriak @yanglbme @debasishbsws

**Review by siriak — CHANGES_REQUESTED** [review]
Please add .java extension to your file, use PascalCase or camelCase as appropriate, and fix PR checks

**Review by adjamalbek — COMMENTED** [review]
updated

File: src/test/java/com/thealgorithms/maths/PlusOneTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        assertEquals(intArray_2, PlusOne.plusOne(intArray_2_test));
+    }
+
+
+      @Test
+    void test_three() {
+        assertEquals(intArray_3, PlusOne.plusOne(intArray_3_test));
+    }
+
+}
```
Comment (debasishbsws) [inline src/test/java/com/thealgorithms/maths/PlusOneTest.java:33]:
```suggestion
int[] intArray1 = new int[]{ 1,2,4 }; 
int[] intArray2 = new int[]{ 4,3,2,2 }; 
int[] intArray3 = new int[]{ 1,0 }; 

int[] intArray1Test = new int[]{ 1,2,3 }; 
int[] intArray2Test = new int[]{ 4,3,2,1 }; 
int[] intArray3Test = new int[]{ 9 }; 

    @Test
    void testOne() {
        assertEquals(intArray1, PlusOne.plusOne(intArray1Test));
    }
    @Test
    void testTwo() {
        assertEquals(intArray2, PlusOne.plusOne(intArray2Test));
    }
    @Test
    void testThree() {
        assertEquals(intArray3, PlusOne.plusOne(intArray3Test));
    }
}
```

**Review by debasishbsws — CHANGES_REQUESTED** [review]
The variables and method name should be in camelCase

**Pull request closed without merging**